include/ZmqLogger.h

```diff
@@ -29,17 +29,17 @@
 #define OPENSHOT_LOGGER_H
 
 
-#include "JuceLibraryCode/JuceHeader.h"
-#include <iostream>
-#include <iomanip>
-#include <fstream>
-#include <stdlib.h>
+// #include "JuceLibraryCode/JuceHeader.h"
```

Comment:
Hm ... this is not pretty. If it's commented out it should be removed. Same for the other similar changes here.